app/views/contacts/index.html.haml

```diff
@@ -1,53 +1,47 @@
 .o-page__title
   %h1= t '.title'
-  - if can_edit_contacts?
-    = link_to new_contact_path, class: 'btn btn-primary ml-auto' do
-      %span.icon-plus
-      Add a Contact
-
+%p Contacts are now managed through their associated user accounts.
 - @prompt = t '.search_prompt'
 .row
   .col-sm-4
     = render 'search_form'
 :ruby
-  columns = ['last_name', 'first_name', 'middle_name', 'email', 'phone', 'role']
```

Comment:
I wonder if we should keep role around for the legacy contacts?

Reply:
I'm removing it from the view since you can't set it anymore.  I'm guessing they should all be blank anyway and the role should come from the contact's participation on the match.  If someone asks to have it back, we can just re-enable it and add it to the user form.